css/common.css

```diff
@@ -0,0 +1,57 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700;900&display=swap');
```

Comment:
Hey Manoj in the session today MAs suggested if we are using anything from outside then instead of imports better to use links in the html header , the situation where this point came up was also a Google font so I guess it's better to have these Google fonts as links in the html header , if it's internal links like importing another CSS file then u can use this way

Reply:
Oh I was not aware about this. Thank you for sharing. I'll make changes and follow the same in future.